apko/BUILD.bazel

```diff
@@ -13,7 +13,7 @@ toolchain_type(
 )
 
 resolved_toolchain(
-    name = "resolved_toolchain",
+    name = "current_toolchain",
```

Comment:
how did we get inconsistent about this naming?

Reply:
copy paste from rules_oci i think.